dashboard/management/commands/h1sync.py

```diff
@@ -27,10 +27,9 @@ def handle(self, *args, **options):
             kwargs['last_activity_at__gt'] = metadata.last_synced_at
 
         listing = h1.find_reports(**kwargs)
-        count = len(listing)
```

Comment:
Our use of `itertools.chain` means that we can't take the `len()` of the chained generator, so I had to get rid of this.